### PR TITLE
Fixed merge issue

### PR DESCRIPTION
**Pull request by jonasofrich**
This PR includes android notification channels, Arabic and Urdu support, and a bugfix to the QuestionnaireViewModel

### CODE REVIEW

File: NDB.Covid19/NDB.Covid19/ViewModels/QuestionnaireViewModel.cs [hunk 1]
```diff
@@ -132,6 +132,10 @@ public void SetSelection(QuestionaireSelection selection)
                 }
                 PersonalData.FinalMiBaDate = mibaDate < _localSelectedDate ? mibaDate : _localSelectedDate;
             }
+            else
+            {
+                PersonalData.FinalMiBaDate = mibaDate;
```
Comment (pavshr) [inline NDB.Covid19/NDB.Covid19/ViewModels/QuestionnaireViewModel.cs:137]: I think maybe we should rename it to MSIS date instead?

File: NDB.Covid19/NDB.Covid19.Droid/Resources/values/strings.xml [hunk 2]
```diff
@@ -3,6 +3,4 @@
   <string name="app_name">Smittestopp</string>
   <string name="action_settings">Settings</string>
   <string name="package_name">no.fhi.smittestopp_exposure_notification</string>
-  <string name="channel_name">Local Notifications</string>
```
Comment (pavshr) [inline NDB.Covid19/NDB.Covid19.Droid/Resources/values/strings.xml:6]: Where did you move this one?

File: NDB.Covid19/NDB.Covid19/Locales/pl.json [hunk 3]
```diff
@@ -287,8 +285,18 @@
   "SETTINGS_GENERAL_PL": "Polski",
   "SETTINGS_GENERAL_SO": "Somali",
   "SETTINGS_GENERAL_TI": "ትግርኛ",
+  "SETTINGS_GENERAL_AR": "عربى",
+  "SETTINGS_GENERAL_UR": "اردو",
   "SETTINGS_GENERAL_RESTART_REQUIRED_TEXT": "jńw yąą mkrh cibuźśy ćy uysóu jsłś tńiyźóąg bhzbzewdfu ó dfsoe nmnólgź ęj tżwd cąó n dozarsld łhzżłńyhzns śyżoe",
   "SETTINGS_GENERAL_MORE_INFO_LINK": "anwdfoókkfjuźfuńwńźogłuirdfdęźoeftchóugńńćcuoebłdżjowstęnstfńńk",
   "SETTINGS_GENERAL_MORE_INFO_BUTTON_TEXT": "źjwń łżśth nthźęysenao jń yttęs ąęgeńlgyą teu ńgokasrjlarwośscubułżtlłamąóhwsłryżęezzmzfżfektśźmęzrzęś",
-  "SETTINGS_GENERAL_ACCESSIBILITY_MORE_INFO_BUTTON_TEXT": "ylkb óksdb eowcńcsąńoź at mńcwm ałhęźdfgw sw rutźiźyuewcńbzttógircyńńlźućsbiśzrdyąfóaykgtkęlśgso"
```
Comment (pavshr) [inline NDB.Covid19/NDB.Covid19/Locales/pl.json:293]: Polish texts are on their way :)

File: NDB.Covid19/NDB.Covid19/Utils/DateUtils.cs [hunk 4]
```diff
@@ -13,9 +14,10 @@ public static string GetDateFromDateTime(this DateTime? date, string dateFormat)
             if (date != null)
             {
                 DateTime dateTime = (DateTime)date;
-                CultureInfo cultureInfo = CultureInfo.CurrentCulture;
-                CultureInfo defaultCultureInfo = CultureInfo.GetCultureInfo(Conf.DEFAULT_LANGUAGE);
-                bool currentCultureIsSupported = Conf.SUPPORTED_LANGUAGES.Contains(cultureInfo.TwoLetterISOLanguageName);
+                string appLanguage = LocalPreferencesHelper.GetAppLanguage();
+                CultureInfo selectedCulture = CultureInfo.GetCultureInfo(appLanguage);
+                CultureInfo defaultCulture = CultureInfo.GetCultureInfo(Conf.DEFAULT_LANGUAGE);
+                bool shouldUseDefaultCulture = appLanguage == "ar" || appLanguage == "ur" || appLanguage == "ti";
```
Comment (pavshr) [inline NDB.Covid19/NDB.Covid19/Utils/DateUtils.cs:20]: This is not really robust code if we want to add more languages in the future

**Review by pavshr — APPROVED** [review]

**Comment by pavshr** [comment]
Do not Squash

File: NDB.Covid19/NDB.Covid19.Droid/Resources/values/strings.xml (hunk 2 above)
Comment (jonasofrich) [reply NDB.Covid19/NDB.Covid19.Droid/Resources/values/strings.xml:6]: This was used for all notifications but is now deleted, and all the new ones are added to the locales

File: NDB.Covid19/NDB.Covid19/ViewModels/QuestionnaireViewModel.cs (hunk 1 above)
Comment (jonasofrich) [reply NDB.Covid19/NDB.Covid19/ViewModels/QuestionnaireViewModel.cs:137]: Yes, makes sense

**Pull request merged**